### PR TITLE
fix: add small More button for small screen [DHIS2-10424]

### PR DESCRIPTION
**Pull request by jenniferarnesen**
Issue was that the More button is together in a button strip containing other buttons. The other buttons are hidden in small screen, but More button is still available, albeit in "small".  This conversion to "small" was being handled by javascript. So there was a lag before the button changed. 

This proposed solution is to have two More buttons and they are hidden or shown using media query.

https://user-images.githubusercontent.com/6113918/107034280-c878c980-67b6-11eb-92ba-0db28f61a403.mov

### CODE REVIEW

File: src/components/TitleBar/ViewTitleBar.js [hunk 1]
```diff
@@ -126,12 +123,21 @@ const ViewTitleBar = props => {
                             <FilterSelector />
                             <span ref={buttonRef}>
                                 <Button
-                                    small={isSmallScreen(width)}
-                                    dataTest="more-button"
+                                    className={classes.moreButton}
```
Comment (martinkrulltott) [inline src/components/TitleBar/ViewTitleBar.js]: No biggie, but to reduce code repetition, parts of the button could be fetched from e.g. `const getMoreButton = className =>` or `const getMoreButtonContent = () =>`

File: src/components/TitleBar/styles/ViewTitleBar.module.css [hunk 2]
```diff
@@ -55,10 +55,14 @@
     color: #888;
 }
 
-.moreButton {
+.moreText {
     margin-left: 5px;
 }
 
+.moreButtonSmall {
+    display: none !important;
```
Comment (martinkrulltott) [inline src/components/TitleBar/styles/ViewTitleBar.module.css]: Can `!important` be avoided by increasing the specificity with `.strip .moreButtonSmall`?

**Review by varl — COMMENTED** [review]
Thanks for adding me, @jenniferarnesen. I think this use case may be common so I'll have it in mind if it can be made easier through UI.

File: src/components/TitleBar/styles/ViewTitleBar.module.css (hunk 2 above)
Comment (jenniferarnesen) [reply src/components/TitleBar/styles/ViewTitleBar.module.css]: Done

File: src/components/TitleBar/ViewTitleBar.js (hunk 1 above)
Comment (jenniferarnesen) [reply src/components/TitleBar/ViewTitleBar.js]: Done

**Review by cooper-joe — APPROVED** [review]

**Review by martinkrulltott — APPROVED** [review]

**Pull request merged**